### ISSUE TITLE
Please add ruby and node

### ISSUE THREAD

**Issue by christiangenco** [holzschu/a-shell#46]
Thank you so much for building this—`a-shell` feels like one of the last missing pieces for me to actually be productive on my iPad.

My primary development environment is in node (lots of frontend React apps) and my primary scripting language is Ruby (I realize python is practically the same thing, but old habits and all that).

This issue would potentially be resolved by a package manager, as in #42.

**Comment by Himself65** [comment]
+1.

I’m nodejs collaborator and also would like to support `node.js`. it would be great if do this. 

for now, `node.js` has official support `JIT-less` mode, I think this will help to support nodejs on `a-Shell`

Refs: https://github.com/nodejs/node/pull/32594

**Comment by zaptrem** [comment]
Any chance of this? Node and NPM opens up a whole world of options and dev tools.

**Comment by therocketforever** [comment]
Ruby would be incredibly valuable to me as an engineer

**Comment by kmf** [comment]
+1 for Ruby!

**Comment by kkk669** [comment]
The WASI port of CRuby has been merged, so it may soon be able to run on a-Shell.

https://bugs.ruby-lang.org/issues/18462

**Comment by tdegrunt** [comment]
Awesome, hopefully this will be available by default.

**Comment by xor256** [comment]
Yes, Ruby is important and irb and lib dev ruby for making a gem or extension
